### DISCUSSION TITLE
Guidance for 2025 term 1

### DISCUSSION THREAD

**Discussion by uddhavphatak** [cncf/mentoring#1311] [Mentee hub]
Hi,
I’m seeking guidance on starting my journey as a mentee under CNCF. I was a successful GSoC contributor in 2024 with the Linux Foundation and am now eager to contribute to CNCF. I have the following questions:

1. What would be a good starting point for me to begin contributing?
2. How can I connect with mentors who can guide me and help address my doubts about the projects?
3. Are there any tips or insights that I might not be aware of but could significantly improve my contributions and growth as a developer?

**Comment by chethanm99** [comment]
Hi Uddhav ,

If you are interested in contributing to any project you can check out it's mailing list or slack channel where you can ask about your doubts. You can also checkout the #mentoring channel under CNCF workspace.

To know more about contributing to CNCF you can checkout : https://contribute.cncf.io/
You can also checkout : https://contribute.cncf.io/contributors/

Hope this helps!
